R/LSgaussianestimation.R

```diff
@@ -290,28 +293,32 @@ LSgaussianestimation   <- function(jaspResults, dataset, options, state = NULL){
 }
 .plotsIndividualGaussianLS         <- function(jaspResults, data, ready, options, type = c("Prior", "Posterior")){
 
-  containerPlots <- .containerPlotsLS(jaspResults, options, "gauss_est", type)
+  containerPlots <- .containerPlotsLS(jaspResults, options, "gaussEst", type)
 
   if(is.null(containerPlots[[paste0("plots",type,"individual")]])){
 
     plotsIndividual <- createJaspContainer()
 
     plotsIndividual$position <- 2
-    plotsIndividual$dependOn(c(.GaussianLS_data_dependencies,
-                               ifelse(type == "Prior", "plotsPriorIndividualCI", "plotsPosteriorIndividualCI"),
-                               ifelse(type == "Prior", "plotsPriorCoverage",     "plotsPosteriorCoverage"),
-                               ifelse(type == "Prior", "plotsPriorLower",        "plotsPosteriorLower"),
-                               ifelse(type == "Prior", "plotsPriorUpper",        "plotsPosteriorUpper")))
+    plotsIndividual$dependOn(c(.dataDependenciesGaussianLS,
+                               ifelse(type == "Prior", "plotsPriorIndividualEstimate",     "plotsPosteriorIndividualEstimate"),
+                               ifelse(type == "Prior", "plotsPriorIndividualEstimateType", "plotsPosteriorIndividualEstimateType"),
+                               ifelse(type == "Prior", "plotsPriorIndividualCI",   "plotsPosteriorIndividualCI"),
+                               ifelse(type == "Prior", "plotsPriorIndividualType", "plotsPosteriorIndividualType"),
+                               ifelse(type == "Prior", "plotsPriorCoverage",       "plotsPosteriorCoverage"),
+                               ifelse(type == "Prior", "plotsPriorLower",          "plotsPosteriorLower"),
+                               ifelse(type == "Prior", "plotsPriorUpper",          "plotsPosteriorUpper"),
+                               if(type == "Posterior") c("plotsPosteriorBF", "plotsPosteriorIndividualPrior", "plotsPosteriorIndividualProportion")))
 
     containerPlots[[paste0("plots",type,"individual")]] <- plotsIndividual
 
 
-    if(all(!ready) || (ready[1] && !ready[2])){
+    if(all(!ready) || (ready["data"] && !ready["priors"])){
```

Comment:
```suggestion
    if (all(!ready) || (ready["data"] && !ready["priors"])) {
```
and same elsewhere for `if`, `else if` , `else`, and `for`